lua/entities/gmod_wire_expression2/base/compiler.lua

```diff
@@ -1868,7 +1868,9 @@ local CompileVisitors = {
 				---@type E2Lambda
 				local f = expr(state)
 
-				if f.arg_sig ~= sig then
+				if not f then
+					state:forceThrow("Trying to call function that doesn't exist!")
```

Comment:
Set the default function to one that throws this error. This is adding overhead to every function call

Reply:
I wanted to do that but then it gives an error for bad parameters instead, which I thought would be unideal.
```ts
let BadFunc = table()[1, function]

BadFunc(1) # Error: Expected void
```
If you think that's worth sacrificing, then that's fine. The error stems from the same root I guess.

Reply:
As a compromise, maybe the error could be thrown *after* `if f.arg_sig ~= sig`? Then it would only be adding that overhead to errors. Compare it to a `nofunction` constant.